### PR TITLE
Add integration tests to pipeline; Add missing spatial audio style to…

### PR DESCRIPTION
**Pull request by kbetl-dlb**
… mocks

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -77,12 +84,22 @@ jobs:
...
       run: |
         ./scripts/run-unmocked-integration-tests-ios.rb
     - name: Run mocked integration tests
       run: |
         ./scripts/run-mocked-integration-tests-ios.rb
+
     - name: Analyze
       run: |
-        flutter analyze
+        flutter analyze
```
Comment (graduad) [inline .github/workflows/ci.yml:105]: I was wondering, why do you delete the last new line?

File: scripts/macos_configure-sdkmanager.sh [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+#
+#chmod +x silent_install.sh
+#
+#sudo ./silent_install.sh
+#
+#kextstat | grep intel
+#
+#sudo kextunload -b com.intel.kext.intelhaxm
+#
+#sudo kextload -b com.intel.kext.intelhaxm
```
Comment (graduad) [inline scripts/macos_configure-sdkmanager.sh:51]: Commented out code. Do we need this?

File: scripts/macos_setup_android_env.sh [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+echo "Setup android"
+
+#lsb_release -a
+
+#sudo apt -y update
+
+# java 8 is required for avdmanager
+#sudo apt-get install openjdk-8-jdk
+
+#sudo apt-get install libgl1-mesa-dev
```
Comment (graduad) [inline scripts/macos_setup_android_env.sh:12]: Commented out code. Do we need it?

**Review by graduad — COMMENTED** [review]
I noticed quite a few lines of commented out code. I have only put two comments about it, but it would be good to remove all the commented out code that we do not need.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (kbetl-dlb) [reply .github/workflows/ci.yml:105]: my mistake

File: scripts/macos_configure-sdkmanager.sh (hunk 2 above)
Comment (kbetl-dlb) [reply scripts/macos_configure-sdkmanager.sh]: I removed it

File: scripts/macos_setup_android_env.sh (hunk 3 above)
Comment (kbetl-dlb) [reply scripts/macos_setup_android_env.sh]: I removed it

File: .github/workflows/ci.yml (hunk 1 above)
Comment (kbetl-dlb) [reply .github/workflows/ci.yml]: revert changes

**Review by graduad — DISMISSED** [review]

**Review by graduad — APPROVED** [review]

**Pull request merged**